x-pack/legacy/plugins/siem/public/components/timeline/search_super_select/index.tsx

```diff
@@ -287,7 +289,11 @@ const SearchTimelineSuperSelectComponent: React.FC<SearchTimelineSuperSelectProp
                 rowHeight: TIMELINE_ITEM_HEIGHT,
                 showIcons: false,
                 virtualizedProps: ({
-                  onScroll: handleOnScroll.bind(null, timelines.length, totalCount),
+                  onScroll: handleOnScroll.bind(
+                    null,
+                    timelines.filter(t => !hideUntitled || t.title !== '').length,
```

Comment:
title can be null/undefined/''
```suggestion
                    timelines.filter(t => !hideUntitled || isEmpty(t.tittle)).length,
```